modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -52,18 +52,11 @@
   }
 
   # wallpaper symlink
-  if $::lsbdistcodename == 'jessie' {
-    $wallpaper = $staff_only ? {
-      true  => 'background-staff.png',
-      false => 'background.png',
-    }
-  } else {
-    $wallpaper = $staff_only ? {
-      true  => 'background-staff.svg',
-      false => 'background.svg',
-    }
+  $wallpaper = $staff_only ? {
+    true  => 'background-staff.svg',
+    false => 'background.svg',
   }
-
+  
```

Comment:
Remove this

Reply:
lol I did this from the github web interface